### PR TITLE
Add Snackbar

### PR DESCRIPTION
**Pull request by Garbee**
What's needed now?

* [ ] Review of doc page content
   * [ ] Better description/introduction
* [ ] Unit Tests
* [ ] Material Design Alignment review
    * I need to work on getting a solid example for @santokii that is static for reviewing the snackbar itself
* [ ] Setup for component page in the doc area. Will discuss structure for this via email with core team.
* [ ] Refactor for new IIFE component setup.

Known issues:
* No animation
    * In the move to using dynamic elements, animating them became problematic so I dropped it. If anyone knows how to handle that, go for it.

### CODE REVIEW

File: src/snackbar/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+&nbsp;2. Add the `mdl-js-snackbar` class to the container.
+
+```html
+<div aria-live="assertive" aria-atomic="true" aria-relevant="text" class="mdl-js-snackbar">
+</div>
+```
+
+> Note: Here we also added some basic aria attributes for accessiblity. Please modify these as-needed for your site.
```
Comment (samccone) [inline src/snackbar/README.md]: Here => In this example we also ...

**Comment by surma** [comment]
@Garbee Is the checklist in this PR supposed to be checked off before merging? Or are those things supposed to be done after the facT?

**Comment by Garbee** [comment]
**BEFORE**. A lot of work still needs to go into this. The only thing that *can* lag is the documentation setup. I'll update the primary comment after lunch to be more detailed.

**Comment by surma** [comment]
Thank you, @Garbee :)

**Comment by Garbee** [comment]
List is updated. Happened way later than lunch... not a good day.

**Comment by freeslugs** [comment]
thanks @Garbee for your contributions! eager to use snackbar in an app - any updates on the progress / expected completion date?

**Comment by hebbet** [comment]
@freeslugs snackbar is currently planned for 1.1 which is scheduled for Oct 15th according to current milestone data.

**Comment by surma** [comment]
> 1.1 is scheduled for Oct 15th

Please keep in mind that this is by no means a hard deadline. It’s something we aim for, but it can change!

**Comment by pho3nixf1re** [comment]
Is this at least usable in an alpha state? For example, could I rip out the additions and drop them into my own project to at least get basic functionality? What known issues should I look out for if I do? I regret having chosen MDL for an app that I'm pretty far along in as it lacks many necessary features, however I really love the direction MDL is taking so I willing to help out getting things tested and working.

**Comment by surma** [comment]
@pho3nixf1re The current state of snackbars is in this PR (and the branch it references). Feel free to use it, but keep in mind that we will not accept issues for this component as long as this hasn’t been merged into master.

**Comment by pho3nixf1re** [comment]
Fair enough!

**Comment by mfonville** [comment]
@pho3nixf1re when you will be building this branch, you'd mind to share the result via e.g. dropbox or drive? I'd be also interested to use it, but lack the resources to do my own build :-/

**Comment by pho3nixf1re** [comment]
@mfonville I'm not forking it. I want to wait on a proper merge so I'm just temporarily ripping the code out of here and directly putting it in my project instead of importing the styles and JS with webpack. Sorry!

**Comment by Garbee** [comment]
So, master rebase failed. Closing this out and will do some more work then make a new PR with a clean history.

**Comment by mfonville** [comment]
@Garbee Would you please mention the new PR in this one when ready? Then we as subscribers can easily track progress.

**Comment by Garbee** [comment]
PR #1519 is the new PR for this.

**Pull request closed without merging**